src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -4444,7 +4444,7 @@ void CalcHeatEmissionReport(EnergyPlusData &state)
     // Water heater and thermal storage
     auto &WaterThermalTank(state.dataWaterThermalTanks->WaterThermalTank);
     for (int iTank = 1; iTank <= state.dataWaterThermalTanks->numWaterThermalTank; ++iTank) {
-        if (WaterThermalTank(iTank).AmbientTempIndicator == WaterThermalTanks::AmbientTempEnum::OutsideAir) {
+        if (WaterThermalTank(iTank).AmbientTempIndicator == WaterThermalTanks::WTTAmbientTemp::OutsideAir) {
```

Comment:
OK, so renamed the enum I assume.  :+1:

Reply:
Yes, just wanted to take the word "Enum" out.